bin/hashbangctl

```diff
@@ -141,6 +143,23 @@ while True:
 
         print('\nPlease press [Enter] to continue')
 
+    elif key == 'H':
+        hosts = [
```

Comment:
Could you fetch this list from the database?

Reply:
should be resolved